src/server/db_slice.cc

```diff
@@ -1023,27 +985,6 @@ void DbSlice::PreUpdate(DbIndex db_ind, PrimeIterator it) {
     ccb.second(db_ind, ChangeReq{it});
   }
 
-  auto* stats = MutableStats(db_ind);
-  if (it->second.ObjType() == OBJ_STRING) {
-    if (it->second.IsExternal()) {
-      // We assume here that the operation code either loaded the entry into memory
```

Comment:
Not sure how this worked .. we did not load the entry into memory before PreUpdate. 
Anyway now we load the data if we got load_external = true for Find commands of db slice, preupdate happens after that and we do not want to to reset external objects if we dont load the data